projects/glib/fuzz_variant_text.dict

```diff
@@ -0,0 +1,29 @@
+value="'"
```

Comment:
It's not necessary to have a `value=` prefix, especially if it's the same for every element. The main purpose of these names to serve as comments for someone who would read the dictionary. Otherwise, just put elements in double quotes.

Reply:
I've been doing this habitually to have AFL compatibility, which I had assumed requires the name. I must've misread that when I read its docs originally. I just re-read the AFL docs and indeed it can be omitted.